examples/scripts/ct_large_projection.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+r"""
+Large-scale CT Projection
+=========================
+
+This example demonstrates using SCICO's X-ray projector on a large-scale
+volume.
+
+"""
+
+import numpy as np
+
+import jax
+
+from scico.examples import create_block_phantom
+from scico.linop.xray import XRayTransform3D
+
+N = 1000
+num_views = 10
+
+in_shape = (N, N, N)
+x = create_block_phantom(in_shape)
+
+det_shape = (N, N)
+
+rot_X = 90.0 - 16.0
+rot_Y = np.linspace(0, 180, num_views, endpoint=False)
+angles = np.stack(np.broadcast_arrays(rot_X, rot_Y), axis=-1)
+matrices = XRayTransform3D.matrices_from_euler_angles(
+    in_shape, det_shape, "XY", angles, degrees=True
+)
+
+
+H = XRayTransform3D(in_shape, matrices, det_shape)
+
+proj = H @ x
+jax.block_until_ready(proj)
```

Comment:
This example seems to be incomplete. Was it intended to display timing information? Also, consider renaming to `ct_projection_large.py`.

Reply:
I added this by accident. Would it be useful to finish it (show timing info and display the result)?

Reply:
Perhaps. It may be better to remove it for now, keeping a private copy to be completed and re-added later.